sites/friday/src/get-started/nextjs/_index.md

```diff
@@ -1,4 +1,518 @@
+  "https://www.{default}/":
+    type: redirect
+    to: "https://{default}/"
+```
+
+Then commit your new files, using the following command:
+
+```bash {location="Terminal"}
+git add .environment .{{% vendor/cli %}}/config.yaml
+git commit -m "{{% vendor/name %}} config files"
+```
+
+[//]: # (TODO this entire section should not exist and be remove in favor of adding this new build hook command in the project:init)
+## Build your Next.js application
+
+Next.js application needs be built during [build hook](/create-apps/app-reference.html#build-hook). To do so, edit your `.{{% vendor/cli%}}/config.yaml` file with the following:
+```yaml {location=".{{% vendor/cli%}}/config.yaml"}
+applications:
+  app:
+    {{< code-link destination="/create-apps/app-reference.html#hooks" text="hooks" title="Hooks allow you to customize your code/environment as the project moves through the build and deploy stages. Click for more information" >}}:
+      {{< code-link destination="/create-apps/app-reference.html#build-hook" text="build" title="The build hook is run after any build flavor.. Click for more information" >}}: |
+        set -eux
+        npm i
+        npm exec next build
+```
+
+Then commit your new files, using the following command:
+
+```bash {location="Terminal"}
+git add .{{% vendor/cli %}}/config.yaml
+git commit -m "build Next.js app during build hook"
+```
+
+## Set project remote
+
+{{< note >}}
+If you used the {{% vendor/name %}} CLI command `{{% vendor/cli %}} project:create` to create your project and your local Git repo has already been initialized, your local source code should already contain a ``.{{% vendor/cli %}}/local/project.yaml`` file. This file contains your `projectId`, and you already have a Git remote repository set to `{{% vendor/cli %}}`.
+You can jump to [deploying your project](#deploy).
+{{< /note >}}
+
+There are slightly different ways to link your local project to your {{% vendor/name %}} project based on the Git workflow you chose for you project as discussed earlier in this guide.
+
+{{< codetabs >}}
++++
+title={{% vendor/name %}} Git repository
++++
+If you host your Next.js source code on an {{% vendor/name %}} Git repository, and you failed to answer `y` (yes) to the question `Set the new project <projectName> as the remote for this repository? [Y/n]` during the ``project:create`` command, you need to let the {{% vendor/name %}} CLI know which linked project you want to deploy to.
+
+To do so, use the {{% vendor/name %}} CLI to set remote project:
+```bash {location="Terminal"}
+{{% vendor/cli %}} project:set-remote <projectId>
+```
+
+This command will add a new remote called `{{% vendor/cli %}}` to your local Git repo as you can see below:
+```bash {location="Terminal"}
+git remote
+{{% vendor/cli %}}
+...
+```
+
+It will also create a new `.{{% vendor/cli %}}/local/project.yaml` file that will contain the given `<projectId>`, to store this info for the {{% vendor/name %}} CLI interaction.
```

Comment:
```suggestion
It will also create a new `.{{% vendor/cli %}}/local/project.yaml` file that will contain the given `<projectId>`, to store this information for the {{% vendor/name %}} CLI interaction.
```